axelor-account/src/main/java/com/axelor/apps/account/service/move/MoveToolService.java

```diff
@@ -423,8 +423,7 @@ public boolean isDebitMoveLine(MoveLine moveLine) {
 
   public String filterJournalPartnerCompatibleType(Move move) {
     Journal journal = move.getJournal();
-
-    if (journal != null && journal.getCompatiblePartnerTypeSelect() != null) {
+    if (journal != null && journal.getCompatiblePartnerTypeSelect() != null && !journal.getCompatiblePartnerTypeSelect().isEmpty()) {
```

Comment:
`journal.getCompatiblePartnerTypeSelect() != null && !journal.getCompatiblePartnerTypeSelect().isEmpty()` can be simplified by `!StringUtils.isEmpty(journal.getCompatiblePartnerTypeSelect())`